### DISCUSSION TITLE
Option to assign Spotify Account to User

### DISCUSSION THREAD

**Discussion by SodaSyrup** [music-assistant/support#2928] [Feature requests and ideas]
Hello, it would be great if there was an option to assign Spotify Providers to specific users, so everyone can listen to their own music on their own speakers at the same time and then also not get song recommendations and all that mixed up. 

I saw a similar post somewhere here, but i wasnt sure if i understood correctly so i thought id make my own and describe it with a bit more simpler terms.

Biggest Benefit is simultaneous listening, access to their own playlist (which i know is possible by simply adding all accounts). and then the biggest one, the song/playlist recommendations, and by that the automatic radio song choices that spotifys algorithm makes, doesnt get mixed up.

**Comment by OzGav** [comment]
This has been added to MA 2.7 and above

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
